docs/input.md

```diff
@@ -219,6 +199,37 @@ description: |
     This is the second line
 ```
 
+#### `example`
+
+The `example` field is optional and can be used to reference a sample dataset
+for the given input, **within** the container. Data examples are a prime source 
+for your users to understand how inputs should look like and be formatted.
+
+```yaml
+example: /samples/input_name.csv
+```
+
+#### `quality`
```

Comment:
The `quality` field is also a good idea, but I'm not sure what we would do with that in practice, is it just a free text or do we want to include checks of appropriate data quality or something like that when running a tool?

Reply:
I have really no idea how we could quality check arbitrary data of any kind, any variable and any format in a meaningful way, whatsoever.
The idea is more aimed at giving the developer of a tool the possibility to highlight data quality considerations. The user can be made aware of them and client applications could implement them as warnings, or hints.